server/controllers/amp-page.js

```diff
@@ -20,6 +20,7 @@ module.exports = (req, res, next) => {
 		analyticsConfig: JSON.stringify(analytics.getJson({req, uuid: req.params.uuid})),
 		overrideBlog: req.query.overrideBlog,
 		lastUpdate: req.query.amp_latest_update_time,
+		ftConsentCookie: req.cookies.FTConsent,
```

Comment:
this cookie will never be set, since it's googlebot that's hitting this endpoint, not the user. i don't know how you'd get around that 😕

Reply:
@quarterto True true. If it's never set though, then we won't be including the targeted info so this still makes us GDPR compliant. And if it's accessed via redirects and we have the cookie then we can include some extra targeting. Google also have an `amp-consent` element but it's not really what we want. The whole point of amp is to make it super fast, not ask for consent again just for a bit of targeted info.

Reply:
yeah if we're happy to get rid of this targeting for ~80% of users then go for it lol

Reply:
@quarterto I'll check with adops but I think amp page ads traffic is minimal. <1% of overall